Vagrantfile

```diff
@@ -40,15 +40,15 @@ Vagrant.configure("2") do |config|
   
   # Python environment, system
   config.vm.provision :shell, :privileged => false, :path => "provision_scripts/install_scipy_stack.sh"
-  config.vm.provision :shell, :privileged => false, :path => "provision_scripts/install_pyshp.sh"
+  #config.vm.provision :shell, :privileged => false, :path => "provision_scripts/install_pyshp.sh"
```

Comment:
Why not just remove these lines and the corresponding script if they are not needed.  We can always resurrect them from version control if they are needed.